Remove printk logging in ebpf

We have some residual printk logging in ebpf code. We should remove them because they might have unknown performance impact.

- https://github.com/kubearmor/KubeArmor/blob/e64ccb7fede861923bb49b6909a25c3860d18288/KubeArmor/BPF/system_monitor.c#L521
- https://github.com/kubearmor/KubeArmor/blob/e64ccb7fede861923bb49b6909a25c3860d18288/KubeArmor/BPF/system_monitor.c#L1140

**Current Behaviour**
If you run ` cat /sys/kernel/debug/tracing/trace_pipe`
You would be flooded with
```
        promtail-17342   [003] d... 616275.580299: bpf_trace_printk: ignoring scope 0 4026532275
        promtail-17342   [003] d... 616275.580526: bpf_trace_printk: ignoring scope 0 4026532275
```

**Expected Behaviour**
No output

@daemon1024 I am having some issues setting up my local development environment. I am following [this](https://github.com/kubearmor/KubeArmor/blob/main/contribution/development_guide.md) guide. 
I have installed vagrant and virtualbox but when I run `make vagrant-up` it gives me this error:
```
❯ make vagrant-up
make: *** No rule to make target 'vagrant-up'.  Stop.
```

--- a/KubeArmor/BPF/system_monitor.c
+++ b/KubeArmor/BPF/system_monitor.c
@@ -518,7 +518,6 @@ static __always_inline u32 drop_syscall(u32 scope)
         u32 *d_on_off_switch = bpf_map_lookup_elem(d_visibility, &scope);
         if (d_on_off_switch)
             if (*d_on_off_switch)
-                bpf_printk("ignoring scope %d %u", scope, okey.pid_ns);
                 default_trace = _IGNORE_SYSCALL;
     }
 
@@ -1137,7 +1136,6 @@ int kprobe__execve(struct pt_regs *ctx)
 
     if (get_kubearmor_config(_ENFORCER_BPFLSM) && drop_syscall(_PROCESS_PROBE))
     {   
-        bpf_printk("dropped");
         return 0;
     }
 

